src/main/java/com/company/demoapplication/ImageProcessor.java

```diff
@@ -43,6 +43,7 @@ void run() {
                 String imageKey;
                 try {
                     imageKey = SqsMessage.deserialize(message.getBody()).imageKey;
+                    System.out.println("Image name: " + getNameFromKey(imageKey))
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error, insert &#34;;&#34; to complete Statement